pkg/duck/sinks.go

```diff
@@ -38,6 +39,15 @@ func GetSinkURI(ctx context.Context, dynamicClient dynamic.Interface, sink *core
 		return "", fmt.Errorf("sink ref is nil")
 	}
 
+	// K8s Services are special cased. They can be called, even though they do not satisfy the
+	// Callable interface.
+	if sink.APIVersion == "v1" && sink.Kind == "Service" {
+		if sink.Namespace != "" {
+			return DomainToURL(ServiceHostName(sink.Name, sink.Namespace)), nil
+		}
+		return DomainToURL(ServiceHostName(sink.Name, namespace)), nil
+	}
+
```

Comment:
I can't comment on it, but the logic below here is wrong.  You need to support the status.address.url :(

Reply:
https://github.com/GoogleCloudPlatform/cloud-run-events/issues/14